node/tessel-export.js

```diff
@@ -453,22 +470,22 @@ Tessel.Port = function(name, socketPath, board) {
 
   this.I2C.enabled = false;
 
-  this.SPI = function SPIInit(format) {
+  this.SPI = function(options) {
     if (port._spi) {
       port._spi.disable();
     }
 
-    port._spi = new Tessel.SPI(format === null ? {} : format, port);
+    port._spi = new Tessel.SPI(options || {}, port);
 
     return port._spi;
   };
 
-  this.UART = function UARTInit(format) {
+  this.UART = function(options) {
     if (port._uart) {
       port._uart.disable();
     }
 
-    port._uart = new Tessel.UART(port, format || {});
```

Comment:
I reviewed T1 APIs and docs, this change will have no effect on any existing code (since all code is using the `tessel.port.A.UART(options)` form.